### PR TITLE
Allow egress security groups

### PR DESCRIPTION
**Pull request by roaet**
RM12220

NCP-1468

Modified unit tests to allow egress as well.

### CODE REVIEW

File: quark/cache/security_groups_client.py [hunk 1]
```diff
@@ -80,7 +80,7 @@ def serialize_rules(self, rules):
                        "source network": source,
                        "destination network": destination,
                        "action": "allow",
-                       "direction": "ingress"}
+                       "direction": direction}
```
Comment (Cerberus98) [inline quark/cache/security_groups_client.py]: Sorry for hardcoding this here instead of making it a constant

**Comment by Cerberus98** [comment]
:+1:

**Comment by jmeridth** [comment]
derp, why isn't travis firing?

**Comment by tr3buchet** [comment]
TRAVIS!!!!!!!

**Pull request merged**